.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.10.0
+    hooks:
+    -   id:   black
+        exclude: migrations
```

Comment:
This isn't needed. This is likely legacy from you copying this from my Django app.